scRNA-seq/04-tag-based_scRNA-seq_processing.Rmd

```diff
@@ -0,0 +1,246 @@
+---
+title: "Processing tag-based single-cell RNA-seq data with Alevin"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+editor_options: 
+  chunk_output_type: inline
+---
+
+**CCDL 2019**
+
+In this notebook, we will be running through the basics of processing raw single-cell RNA-seq data.
+
+We will be using a tag-based scRNA-seq sample from the [*Tabula Muris* project](https://www.nature.com/articles/s41586-018-0590-4).
+This dataset is made of 20 mouse organs that were sequenced using 10X Genomics Chromium single cell sequencing methods.
+For 10X Genomics scRNA-seq data, cells are separated by emulsion/droplets, and individual  cells are given barcodes (often abbreviated 'CB' in documentation).
+These data also have [Unique Molecular Identifiers (UMIs)](http://www.nature.com/doifinder/10.1038/nmeth.2772) which allow us to examine PCR amplification errors and biases.
+
+**Relative workflow for tag-based data**
+![](diagrams/tag-based_1.png)
+
+## Step 0: About the data
+
+We obtained these data from Tabula Muris project's [Figshare](https://figshare.com/projects/Tabula_Muris_Transcriptomic_characterization_of_20_organs_and_tissues_from_Mus_musculus_at_single_cell_resolution/27733).
+The bam files that were on Figshare were converted to `fastq` files using
+[`bamtofastq`](https://support.10xgenomics.com/docs/bamtofastq) from 10X Genomics.
+We will process a `fastq` file from mouse bladder for this as an example.  
+To limit the amount of time this takes to run in the context of this workshop,
+we are only running part of the sample's reads.
+
+*Note*: depending on the state of the data you are working with, ie. if you have
+a `.bcl` file, you will need to use [`cellranger mkfastq`](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/mkfastq) to make fastq files from this.
+However, most public data is available in `fastq` format, so that is where we will start.
+
+
+## Step 1: Checking your directories
+
+If you have opened the `scRNA-seq.Rproj` file, your Terminal should already be set to the `scRNA-seq` directory, but it is worth checking with the `pwd` command in the Terminal 
+(or by looking at the path shown in the command prompt or at the top of the Terminal pane). 
+If you are in a different directory, we will want to use `cd` to change to the correct directory. 
```

Comment:
```suggestion
If you are in a different directory, we will want to use `cd` to change to the `training-modules/scRNA-seq` directory. 
```
In case they forgot from a sentence ago.